.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@5
-  slack: circleci/slack@4.4.2
+  hmpps: ministryofjustice/hmpps@6
```

Comment:
Now depends on the default docker version provided by circle instead of a static one that was provided by the hmpps orb as recommended by circle.

The [breaking change](https://github.com/ministryofjustice/hmpps-circleci-orb/blob/main/release-notes/6.0.0.md) is to remove any **docker_version** in our config, which we don't have any anyways